orchagent/orchdaemon.cpp

```diff
@@ -566,6 +566,17 @@ bool OrchDaemon::warmRestoreAndSyncUp()
         o->postBake();
     }
 
+    /*
+     * MIRROR ACL rules depend on mirror sessions, which won't be up after warm
+     * reboot until after the postBake step has finished. So, we need to give
+     * AclOrch one more round to resolve any rules that are waiting for mirror
+     * sessions.
+     *
+     * TODO: Implement a more generic mechanism for handling these types of
+     * hard dependencies.
+     */
+    gAclOrch->Orch::doTask();
```

Comment:
You can overwrite postBake in aclOrch

Reply:
We considered that approach too, but the main concern is that the ACL rule operations must be processed strictly after the mirror sessions are created, otherwise the warm reboot validation will fail and orchagent will crash.

We could enforce this through the ordering of the orchs in the orch list, but putting the code here after postBake is more explicit and it's less likely to be re-arranged in the future and cause the warm reboot behavior to regress.

We could also do some sort of retry for postBake, but that is a bigger structural change than this which is beyond the scope of this bug fix.